typings/jsdoc.namespace.js

```diff
@@ -14,10 +14,16 @@ module.exports = {
       }
     },
     newDoclet: ({ doclet }) => {
-      if (doclet.undocumented || doclet.memberof || !kinds.includes(doclet.kind)) return;
+      if (
+        doclet.undocumented ||
+        doclet.memberof ||
+        !kinds.includes(doclet.kind) ||
+        (doclet.kind === 'namespace' && doclet.longname === 'CodeceptJS')
+      )
+        return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected no linebreak before this statement. (nonblock-statement-body-position)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4500224)